garglk/garglk.h

```diff
@@ -32,6 +32,10 @@
 
 #include "gi_dispa.h"
 
+#ifdef GARGLK_USESDL
+#include "SDL/SDL_timer.h"
```

Comment:
One last thing here: I just tried a Windows build, and it fails because the include path points directly to the SDL directory, meaning that `SDL_timer.h` has to be included directly, not using the `SDL/` prefix.  This seems to be the proper way to do it on Unix systems too, as `pkg-config sdl --cflags` spits out `-I/usr/include/SDL`.

I'd also request using angle brackets instead of quotes since SDL is nominally a system library, even though on Windows it's being included directly, e.g.:

```c
#include <SDL_timer.h>
```
The test program looked good, so with this one change, I'll merge it in.

Reply:
Fixed and squashed again.